text/0119-instance-options.md

```diff
@@ -0,0 +1,454 @@
+- [Snapshots](#schedule-snapshot-creation)
+- [Schedule snapshot creation](#schedule-snapshot-creation)
+- [Snapshot destination](#snapshot-destination)
+- [Snapshot interval](#snapshot-interval)
+- [Import snapshot](#import-snapshot)
+- [Ignore missing snapshot](#ignore-missing-snapshot)
+- [Ignore snapshot if DB exists](#ignore-snapshot-if-db-exists)
+- [Memory usage](#memory-usage)
+- [Indexing jobs](#indexing-jobs)
+- [SSL configuration](#ssl-authentication-path)
+- [SSL authentication path](#ssl-authentication-path)
+- [SSL certificates path](#ssl-certificates-path)
+- [SSL key path](#ssl-key-path)
+- [SSL OCSP path](#ssl-ocsp-path)
+- [SSL require auth](#ssl-require-auth)
+- [SSL resumption](#ssl-resumption)
+- [SSL tickets](#ssl-tickets)
+
+#### Database path
+
+**Environment variable**: `MEILI_DB_PATH`
+**CLI option**: `--db-path`
+**Default value**: `"data.ms/"`
+**Expected value**: a filepath
+
+Designates the location where database files will be created and retrieved.
+
+#### Environment
+
+**Environment variable**: `MEILI_ENV`
+**CLI option**: `--env`
+**Default value**: `development`
+**Expected value**: `production` or `development`
+
+Configures the instance's environment. Value must be either `production` or `development`.
+
+`production`:
+
+- Setting a master key is **mandatory**
+- The search preview interface is disabled
+
+`development`:
+
+- Setting a master key is **optional**
+- Search preview is enabled
+
+#### HTTP address & port binding
+
+**Environment variable**: `MEILI_HTTP_ADDR`
+**CLI option**: `--http-addr`
+**Default value**: `"127.0.0.1:7700"`
+**Expected value**: an HTTP address and port
+
+Sets the HTTP address and port Meilisearch will use.
+
+#### Master key
+
+**Environment variable**: `MEILI_MASTER_KEY`
+**CLI option**: `--master-key`
+**Default value**: `None`
+**Expected value**: an alphanumeric string
+
+Sets the instance's master key, automatically protecting all routes except [`GET /health`](/reference/api/health.md). This means you will need an API key to access endpoints such as `POST /search` and `GET /documents`.
+
+You must supply an alphanumeric string when using this option.
+
+Providing a master key is mandatory when `--env` is set to `production`; if none is given, then Meilisearch will throw an error and refuse to launch.
+
+If no master key is provided in a `development` environment, all routes will be unprotected and publicly accessible.
+
+#### Disable analytics
+
+**Environment variable**: `MEILI_NO_ANALYTICS`
+**CLI option**: `--no-analytics`
+**Default**: Enabled
+
+⚠️ This command-line option does not take any values. Assigning a value will throw an error.
+
+Deactivates Meilisearch's built-in telemetry when enabled.
+
+### Dumps destination
+
+**Environment variable**: `MEILI_DUMPS_DIR`
+**CLI option**: `--dumps-dir`
+**Default value**: `dumps/`
+**Expected value**: a filepath pointing to a valid directory
+
+Sets the directory where Meilisearch will create dump files.
```

Comment:
```suggestion
Sets the directory where Meilisearch will create dump files. If the directory does not exist when a dump is generated it will be created.
```